lib/cn/kvset_split.c

```diff
@@ -399,6 +397,13 @@ kblocks_split(
     return err;
 }
 
+struct vgroup_split_metadata {
+    bool overlaps;
+    bool accessed;
+    uint16_t vblk_idx;
+    uint offset;
```

Comment:
This is a `uint` because this is what I get back from the `kvset_iter_next_vref` API.